src/physics/em/detail/MollerBhabhaInteractor.i.hh

```diff
@@ -57,7 +57,9 @@ CELER_FUNCTION MollerBhabhaInteractor::MollerBhabhaInteractor(
 template<class Engine>
 CELER_FUNCTION Interaction MollerBhabhaInteractor::operator()(Engine& rng)
 {
-    if (secondary_energy_cutoff_ >= inc_energy_)
+    if ((secondary_energy_cutoff_ >= 0.5 * inc_energy_
+         && inc_particle_is_electron_)
+        || (secondary_energy_cutoff_ >= inc_energy_))
```

Comment:
Isn't this analogous to first argument to the `max` expression in the constructor (but with inc_e instead of cutoff)? I think it would be more logically expressive as:
```suggestion
    if (inc_energy_ * real_type(inc_particle_is_electron_ ? 0.5 : 1) <= secondary_energy_cutoff_)
```

Reply:
Yep that's better!